ansible/playbook/README.md

```diff
@@ -0,0 +1,39 @@
+---
+title:
+subtitle:
+---
+
+# Hetzner playbooks
+
+## hetzner-init-context
+
+Initializes local hcloud context.
+
+Parameters:
+
+| Variable | Default | Meaning |
+| --- | --- | --- |
+| hetzner_context_name |  | context name |
+| hetzner_token |  | The token to register with Hetzner. |
+
+```bash
+$ ansible-playbook playbook/hetzner-create-hcloud-server.yml
+```
+
+## create hcloud server
+
+Parameters:
+
+| Variable | Default | Meaning |
+| --- | --- | --- |
+| vm_name |  | Name of the VM to be created at hetzner. |
```

Comment:
When the column default id empty, that means that no default exists within the role ? 
Can we also specify what are the mandatory parameters for the different vars ?

Reply:
I have doubts regarding this column, The document might be better of without it. There are no mandatory parameters, one can prevent variables from being evaluated with if defined. IMO all prompts are required, optional variables aren't in prompts.

Reply:
I will review the table after the merge and this is certainly something that we could do for the documentation in general